### ISSUE TITLE
Create better post parser 

### ISSUE THREAD

**Issue by NowanIlfideme** [open-mafia/ml_bay12_classification#3]
Use Nowan's project as an example implementation. Add Date and Title parsing, maybe quote information.

**Comment by ilyalasy** [comment]
As an example you mean https://github.com/open-mafia/kaggle_mafia_scraper?

**Comment by NowanIlfideme** [comment]
Yes. Feel free to copy over the code and/or rewrite it as you wish. I believe every message can be uniquely identified by the board.thread number and the post's order in the thread - we should probably use those as unique keys in our tables, too.
